pkg/virt-controller/watch/vm.go

```diff
@@ -2769,6 +2769,13 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 		syncErr = &syncErrorImpl{fmt.Errorf("Error encountered while storing Instancetype ControllerRevisions: %v", err), FailedCreateVirtualMachineReason}
 	}
 
+	// Once we have ControllerRevisions make sure they are fully up to date before proceeding
+	if err := c.instancetypeMethods.Upgrade(vm); err != nil {
```

Comment:
Inline check above too, to be consistent? (boy scout rule)

Reply:
Follow up.

Reply:
ACK